.eslintrc

```diff
@@ -24,6 +24,10 @@
     ],
     "prettier/prettier": ["error"],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/jsx-props-no-spreading": 1,
```

Comment:
As discussed, understand the reason for putting the overrides in to avoid the scope getting too big.  Will look to remove them soon as time allows though.